github/issues.go

```diff
@@ -47,6 +47,7 @@ type Issue struct {
 	// TextMatches is only populated from search results that request text matches
 	// See: search.go and https://developer.github.com/v3/search/#text-match-metadata
 	TextMatches []TextMatch `json:"text_matches,omitempty"`
+	Locked      *bool       `json:"locked,omitempty"`
```

Comment:
I thought there was some compatibility related reason to keep adding on to the end instead of in the middle, but I'll switch it up